lib/temperature_monitor/temperature_handler.ex

```diff
@@ -0,0 +1,91 @@
+defmodule TemperatureMonitor.TemperatureHandler do
+  use GenServer
+
+  def start_link(stash_pid) do
+    {:ok, _pid} = GenServer.start_link(__MODULE__, stash_pid, name: __MODULE__)
+  end
+
+  def init(stash_pid) do
+    filelist = TemperatureMonitor.Stash.get_value stash_pid
+    monitored_paths = []
```

Comment:
I'll start storing this in the `Stash` next.  Just wanted to get this out there.